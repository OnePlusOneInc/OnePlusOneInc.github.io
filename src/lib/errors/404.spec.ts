import Component from "./404.svelte";
import { render } from "@testing-library/svelte";

describe("404", () => {
  it("should display a 404 page", () => {
    const result = render(Component)
    expect(result.container).toMatchSnapshot()
  });
})