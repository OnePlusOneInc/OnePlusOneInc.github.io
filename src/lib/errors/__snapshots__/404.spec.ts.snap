// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`404 > should display a 404 page 1`] = `
<div>
  <div
    class="h-screen p-4 flex flex-col flex-nowrap align-middle justify-center items-center"
  >
    <img
      alt=""
      class="w-32"
      src="/src/lib/assets/404.svg"
    />
     
    <h1
      class="font-geist font-medium text-4xl sm:text-5xl text-primary-light dark:text-primary-dark mt-6 mb-4"
      type="heading"
    >
      Not found
    </h1>
     
    <h1
      class="font-inter font-normal text-sm sm:text-base text-primary-light dark:text-primary-dark"
      type="copy"
    >
      The page you're looking for doesn't exist
    </h1>
  </div>
</div>
`;
